### ISSUE TITLE
Onboarding guide defender roles outdated

### ISSUE THREAD

**Issue by adonm** [wagov/wasocshared#565]
https://soc.cyber.wa.gov.au//onboarding/#221-tier-0-azure-ad-group-defender-for-endpoint-roles - this page needs review and correction aligned with updated endpoint roles (maybe worth just removing for now until B2B tenant sync + CI/CD content deployment completed) @DGovEnterprise

**Comment by DGovEnterprise** [comment]
@adonm Agreed. Can obfuscate (remove) for the time being until the MTO + Unified experience onboarding process is finalised.
